crossing-repair.lic

```diff
@@ -51,23 +54,27 @@ class CrossingRepair
     @equipment_manager.wear_equipment_set?('standard')
     @equipment_manager.empty_hands
 
-    @skip_bank = settings.sell_loot_skip_bank
+    # @skip_bank = settings.sell_loot_skip_bank
     amount, denom = settings.sell_loot_money_on_hand.split(' ')
```

Comment:
Did you mean to comment this out or was that for testing?

Reply:
Will fix my commenting up this evening.